yarn.lock

```diff
@@ -1290,40 +1290,34 @@
     to-fast-properties "^2.0.0"
 
 "@backstage/core@^0.2.0":
-  version "0.3.0"
+  version "0.2.0"
+  resolved "https://registry.npmjs.org/@backstage/core/-/core-0.2.0.tgz#543246b2d87563c9aa4d9fb96e40fdfc7e827520"
+  integrity sha512-75m2u3FoUngBOvt9l65xZcYTzzB+49OXpY1A9VNFUR1+jMs3cL/0HDfByQV2H0xXaHzMngQ8C5u/sWhkQsij1w==
   dependencies:
     "@backstage/config" "^0.1.1"
-    "@backstage/core-api" "^0.2.1"
-    "@backstage/theme" "^0.2.1"
+    "@backstage/core-api" "^0.2.0"
+    "@backstage/theme" "^0.2.0"
```

Comment:
The lock file looks wrong. These versions shouldn't be downgraded and the dependencies on d3, qs, ... are lost?